### ISSUE TITLE
[Wordpress] Support for IPv6 using /opt/bitnami/bncert-tool

### ISSUE THREAD

**Issue by DevMude** [bitnami/vms#1387]
### Describe your suggestion as much as you can

As stated in the [documentation](https://docs.bitnami.com/aws/faq/administration/generate-configure-certificate-letsencrypt/):

> IMPORTANT: The Bitnami HTTPS Configuration Tool does not support IPv6 addresses or load balancers/CDNs yet. If you use IPv6 addresses, please disable them before proceeding. If you use a load balancer or CDN, please refer to the [alternative approach](https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/#alternative-approach).

AWS is [introducing a new charge for IPv4 addresses](https://aws.amazon.com/blogs/aws/new-aws-public-ipv4-address-charge-public-ip-insights/). This will amount to $3.60/mo per public IPv4 address in use.

Would save users lots of $$$ to have the bncert-tool support IPv6.

**Comment by gongomgra** [comment]
Hi @DevMude,

Thanks for reporting this! I have created an internal task for our team to update the bncert tool. We will post any update here. I will add the on-hold tag as well.

**Comment by gongomgra** [comment]
@DevMude while we are working on our bncert-tool, I just wanted to clarify that [Let's Encrypt officially supports IPv6](https://letsencrypt.org/docs/ipv6-support/), so in case you (or anyone else in our community) are in a hurry, you should be able to generate your own SSL certificates for IPv6-based domains using official Let's Encrypt tools. After that, you can configure your web-server to use the newly generated certificates as described in the guide below

https://docs.bitnami.com/aws/how-to/generate-install-lets-encrypt-ssl/#step-3-configure-the-web-server-to-use-the-lets-encrypt-certificate

**Comment by gongomgra** [comment]
HI @DevMude,

Just a quick note to let you know we have created a new PR with the required changes in the bncert-tool at https://github.com/bitnami/bncert/pull/17

**Comment by gongomgra** [comment]
Hi @DevMude,

Just a quick note to let you know we have released a new bncert-tool asset (version 2.0.0) including support for IPv6-based domains. The tool should detect there is a new version and ask you to auto-update it from current execution. Alternatively, you can download it from the link below

```
https://downloads.bitnami.com/files/bncert/latest/bncert-linux-x64.run
```

Let us know if it works for you now. Hope it helps!
